usagov/vote-gov-drupal
# VOTE-958 Inline alert template and styling

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-958

## Description

Create a new template for the inline alert block and inherit styles from USWDS.

## Deployment and testing

### Pre-deploy

1. lando retune

### Post-deploy

1. Enable the existing inline alert block (Outdated message) http://vote-gov.lndo.site/admin/content/block

### QA/Test

1. Visit http://vote-gov.lndo.site/nv/register/colorado
2. Confirm the inline alert content displays with the default USWDS styling

## Checklist for the Developer




## Checklist for the Peer Reviewers
